karapace/schema_models.py

```diff
@@ -180,6 +183,50 @@ def schema(self) -> Draft7Validator | AvroSchema | ProtobufSchema:
         return parsed_typed_schema.schema
 
 
+class AvroMerge:
+    def __init__(self, schema_str: str, dependencies: Mapping[str, Dependency] | None = None):
+        self.schema_str = json_encode(json_decode(schema_str), compact=True, sort_keys=True)
```

Comment:
Is there not a more efficient way to do this than decoding then re-encoding? It looks like this is just trying to remove  spaces and re-order the output right?

Reply:
It is a sanity check.  See `schema_models.py` parse_avro_schema_definition() function is used this way similarly.